packages/docs/page-config/ui-elements/image/index.ts

```diff
@@ -19,6 +19,13 @@ export default definePageConfig({
     block.example("SrcSet"),
     block.example("Lazy"),
 
+    block.headline('image.imagesInNuxt.title'),
```

Comment:
It is not really image in nuxt. The same way images must be imported in vite spa. You need to mention that "`va-image` behaves the same as img tag, so if you need to use image not form `/public` directory you will need to import it, so bundler like vite or webpack will create a correct img url."